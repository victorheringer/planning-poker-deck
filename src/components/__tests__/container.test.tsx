import { render } from "@testing-library/react";
import Container from "../container";

describe("Container", () => {
  it("should render correctly", () => {
    const tree = render(<Container />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
