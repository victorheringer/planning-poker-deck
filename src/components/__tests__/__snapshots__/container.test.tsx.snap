// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL cFvMCp"
  />
</DocumentFragment>
`;
